about.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>We Care</title>
+    <link rel="icon" type="image/x-icon" href="/img/icon.ico" />
+    <link rel="stylesheet" href="/css/reset.css">
+    <link rel="stylesheet" href="/css/waterfall.css">
+</head>
+<body>
+<div id="header">
+    <h1>
+        <a href="/">We Care</a>
+       <div class="fright menu-links">
+            <a href="/">Campaigns</a>
+            <span>&nbsp;&nbsp;|&nbsp;&nbsp;
+            <a href="/ngos">NGOs</a>
+            <span>&nbsp;&nbsp;|&nbsp;&nbsp;
+            About
+             
+        </div>
+        
+    </h1>
+    <div class="fclear"></div>
+</div>
+
+<div >
+            <h1 style= "padding: 0.8em; background-color: #E5E4E2;">About Us</h1>
```

Comment:
We tried keeping the styling content into a separate CSS file and including that stylesheet into the HTML file, but that wasn't working for us. So we moved on with inline CSS.